modules/ezcontent_node/modules/ezcontent_landing_page/updates_checklist.yml

```diff
@@ -0,0 +1,8 @@
+8.x-1.0:
+    '#title': 'Update to version 8.x-1.0'
+    '#description': '<h3>These are the new features for this release:</h3>'
+    ezcontent_landing_page_update_8001:
+        '#title': 'Remove paragraph content inline block option from layout.'
+        '#description': '<p>This configuration update will update site configuration to newly provided configuration</p>'
+        '#description_successful': '<p>Configuration is successfully updated.</p>'
+        '#description_failed': '<p><strong>Update of configuration has failed.</strong><p>'
```

Comment:
We could avoid this as per my comment: https://github.com/srijanone/ezcontent/pull/332/files#r580805147